concordia/environment/scenes/contrib/scene_generator.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 DeepMind Technologies Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from datetime import datetime
+from collections.abc import Sequence
+import dataclasses
+
+from concordia.associative_memory import formative_memories
+from concordia.language_model import language_model
+from concordia.typing.scene import SceneTypeSpec, SceneSpec
+
+@dataclasses.dataclass
```

Comment:
Why make this a dataclass? It has a function and no data. I would think in this case that it would be more appropriate to make it a regular class.

Reply:
made a regular class